app/(main)/home/CopyTextHeader.tsx

```diff
@@ -1,13 +1,11 @@
 export default function CopyTextHeader() {
   return (
-    <h1
-      className="w-[100%] max-w-[1018px] self-center p-[30px]
-       text-left text-[35px] font-bold leading-[40px] text-white md:p-8
-       md:pt-14 md:text-[50px] md:leading-[70px]"
-    >
-      The step-by-step
-      <span className="text-coursePolicyLightGreen"> AI policy generator</span>
-      <span> for </span>university course instructors.
+    <h1 className="mt-[76px] w-[100%] max-w-[1000px] text-[40px] font-bold text-white ">
```

Comment:
Probably don't need to set `width:100%` here as it's already a block level element, and they default to 100% width.